src/_pytest/runner.py

```diff
@@ -224,7 +224,8 @@ def __repr__(self):
         if self.excinfo:
             status = "exception: %s" % str(self.excinfo.value)
         else:
-            status = "result: %r" % (self.result,)
+            result = getattr(self, "result", "<NOTSET>")
+            status = "result: %r" % (result,)
```

Comment:
@RonnyPfannschmidt 
I think comap.NOTSET is not needed here?!
(as outlined in https://github.com/pytest-dev/pytest/pull/3560#issuecomment-396313096)